modules.py

```diff
@@ -203,6 +230,7 @@ def fits2aperture(self):
 
         self.array_size = np.shape(self.a)[0]
         self.name = 'aperture'
+        logging.info("'aperture.fits' loaded, image size=%s" % self.array_size)
```

Comment:
this logging message should be updated given the 'input_file' variable value instead of the fixed value 'aperture.fits'